pages/home/glossary.mdx

```diff
@@ -87,9 +87,9 @@ A 'user' is your end-user, the person who is using your application or agent.  U
 
 Monthly Active Users are the unique end-users (counted by `user_id`) who have executed a tool in your app/agent within the past month.  If the same `user_id` calls a tool multiple times in the same month, or executes multiple tools, it is only counted once.
 
-### Authentication Starts (Auth Starts)
+### User Authentications (User Auths)
 
-Authentication Starts are the number of unique users (by `user_id`) who have started the authentication flow for a  authentication provider.  This is intended to be a proxy for the number of new users who are using your agent.  The same user authenticating to multiple toolkits will be counted multiple times (e.g. once for Slack and once for Google).  We also count the act of elevating permissions to a user who has already authenticated to a toolkit (e.g. adding a "write" scope when they previously only had a "read" scope).
+User Authentications are the count of permission-level scopes granted to a user or service account that have never been granted before on our platform, including previously held scopes that were deleted and re-granted. The same user authenticating to multiple toolkits will be counted multiple times (e.g. once for Slack and once for Google). We also count the act of elevating permissions to a user who has already authenticated to a toolkit (e.g. adding a "write" scope when they previously only had a "read" scope).
```

Comment:
The reads like we are counting each scope, but in reality we are counting each 'new or updated token'